BenchPress/Helpers/BenchPress.Azure/ActionGroup.Tests.ps1

```diff
@@ -0,0 +1,34 @@
+﻿BeforeAll {
+  Import-Module $PSScriptRoot/ActionGroup.psm1
+  Import-Module Az
+}
+
+Describe "Get-ActionGroup" {
+  Context "unit tests" -Tag "Unit" {
+    BeforeEach {
+      Mock -ModuleName ActionGroup Get-AzActionGroup{}
+    }
+
+    It "Calls Get-AzActionGroup" {
+      Get-ActionGroup -ActionGroupName "agn" -ResourceGroupName "rgn"
+      Should -Invoke -ModuleName ActionGroup -CommandName "Get-AzActionGroup" -Times 1
+    }
+  }
+}
+
+Describe "Get-ActionGroupExist" {
+  Context "unit tests" -Tag "Unit" {
+    BeforeEach {
+      Mock -ModuleName ActionGroup Get-ActionGroup{}
+    }
+
+    It "Calls Get-ActionGroup" {
+      Get-ActionGroupExist -ActionGroupName "agn" -ResourceGroupName "rgn"
```

Comment:
NIT: Out of curiosity, are these built in cmdlets whose names we can't change? I noticed the noun-verb combos do not having 'exists'.

Reply:
I'm not sure that I understand the question. PowerShell uses Verb-Noun, but the test is basically "when Get-ActionGroupExist is called, it calls Get-ActionGroup"

The cmdlets had already been created previously. There is issue #72 that should address cmdlet naming.